src/v2/Apps/Artist/ArtistApp.tsx

```diff
@@ -106,7 +106,7 @@ export const ArtistApp: React.FC<ArtistAppProps> = props => {
               )
             )}
 
-            <Box minHeight="30vh">{children}</Box>
+            {children}
```

Comment:
There were a lot of competing heights and margins on the very top rail of the artist page and this seems to be related. It's too tall with this in place and I'm not sure why this would be necessary.